code-challenges.js

```diff
@@ -8,54 +8,185 @@
+
 // Set one:
 const fruit1 = "apple"
 const fruit2 = "banana"
 
+if(fruit1.length > fruit2.length) {
+    console.log(`${fruit1}`)
+} else if(fruit2.length > fruit1.length) {
+    console.log(`${fruit2}`)
+} else {
+    console.log("Better Luck in Vegas")
+}
+//Output: banana
+
+//I had to refer back to the example provided in the syllabus under conditional statement examples. I know this was the instructor example but I could use the practice. 
+
 // Set two:
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+if(fruit3.length > fruit4.length) {
+    console.log(`${fruit3}`)
+} else if(fruit4.length > fruit3.length) {
+    console.log(`${fruit4}`)
+} else {
+    console.log("Better Luck in Temecula")
+}
+//Output: cherry
+
+//I copied and pasted the code from the fruit1/fruit2 function and changed the numbers to 3 & 4 to match the provided variables. 
+
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
 
+//problem: given a variable with a certain temperature, create a function to return whether the provided variable (temperature) is below, above or at boiling point. Boiling point is 212 degrees Fahrenheit.
+//function expression: if a temperature is less than 212 then the return should be "<temp> is below boiling point". if a temperature is greater than 212 the return should be "<temp> is above boiling point". if a temperature is equal to 212 then the return should be "212 is at boiling point."
+//set parameters: 
+    //boiling point num === 212
+    //temp < 212 return "<temp> is below boiling point"
+    //temp > 212 return "<temp> is above boiling point"
+    //temp = 212 return "<temp> is at boiling point"
+//make a decision: use if and else if statements with <.>.= when making the function for the three possible returns.
+//return should be one of the three string interpolations of below, above or at boiling point with the temperature listed. 
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+if(temp1 < 212) {
+    console.log(`${temp1} is below boiling point`)
+}if(temp2 > 212) {
+    console.log(`${temp2} is above boiling point`)
+}if(temp3 === 212) {
+    console.log(`${temp3} is at boiling point`)
+}
+
+//Output: 42 is below boiling point
+//350 is above boiling point
+//212 is at boiling point
+
+// I had to keep swtiching the conditional statements but I eventually got the ouput that I wanted. I originally had if and else if statements because I was following the example in the syllabus. I'm still trying to understand the differences between them and how and when to use them. I can break down the function in words but putting it together in code has been a struggle for me since jumpstart. 
```

Comment:
This is pretty normal at the beginning.  We certainly don't expect you to memorize all of this within the first 2 weeks.  Instead of using multiple if statements, you should be able to use if, else if, and else statements for this.  Also, just to clarify, conditionals are not functions and it looks like your code is not set up as a function.  Just remember that a function is a reusable block of code that will have the () => {} syntax that is defined by a variable declaration.   The conditional is a code block that you can place inside a function.